hack/netpol/pkg/utils/k8s_util.go

```diff
@@ -0,0 +1,276 @@
+package utils
+
+import (
+
+	//	"context"
+	"bytes"
+	"fmt"
+	"k8s.io/client-go/rest"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	v1net "k8s.io/api/networking/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/tools/remotecommand"
+)
+
+type Kubernetes struct {
+	podCache  map[string][]v1.Pod
+	ClientSet *kubernetes.Clientset
+}
+
+func NewKubernetes() (*Kubernetes, error) {
+	clientSet, err := Client()
+	if err != nil {
+		return nil, errors.WithMessagef(err, "unable to instantiate kube client")
+	}
+	return &Kubernetes{
+		podCache:  map[string][]v1.Pod{},
+		ClientSet: clientSet,
+	}, nil
+}
+
+func (k *Kubernetes) GetPods(ns string, key, val string) ([]v1.Pod, error) {
```

Comment:
could you add some doc string here and for other major func?.. i assume you are retrieving pods relevant to tests